tests/test_index_order.py

```diff
@@ -37,6 +37,8 @@
     "hp_comware",
     "hp_procurve",
     "huawei_vrp",
+    "ipinfusion",
```

Comment:
```suggestion
```
is there a reason for having `ipinfusion`? It looks like `ipinfusion_ocnos` should be sufficient